VMWare/vmware-esxi/tests/AUTH_SESSION_opened.json

```diff
@@ -0,0 +1,33 @@
+{
+  "input": {
+    "sekoiaio": {
+      "intake": {
+        "dialect": "vmware-esxi",
+        "dialect_uuid": "2b13307b-7439-4973-900a-2b58303cac90"
+      }
+    },
+    "message": "pam_unix(sshd:session): session opened for user root by (uid=0)"
+  },
+  "expected": {
+    "message": "pam_unix(sshd:session): session opened for user root by (uid=0)",
+    "observer": {
+      "vendor": "VMware",
+      "product": "ESXi"
+    },
+    "event": {
+      "kind": "event",
+      "category": [
+        "session"
+      ],
+      "type": [
+        "info"
```

Comment:
For this event, a more suitable `event.type` will be `["start"]` as it represent the start of a session.